src/Controller/roleController.js

```diff
@@ -0,0 +1,69 @@
+const Role = require("../Models/Role");
+
+const getAllRoles = (req, res, next) => {
+  Role.find({})
+    .populate("permissions")
+    .then(
+      (roles) => {
+        res.status(200).json(roles);
+      },
+      (err) => next(err)
+    )
+    .catch(() => next(new Error("Cannot get all Role")));
```

Comment:
Error message should be `Cannot get roles`. The current `Cannot get all Role` is grammatically incorrect as it's missing an s at the end of role and if it was correct, it could as well imply that some roles can be returned, just not all roles.